Example/Auth/Tests/FIRPhoneAuthProviderTests.m

```diff
@@ -393,8 +393,12 @@ - (void)testVerifyPhoneNumberUIDelegate {
     // `callbackMatcher` is at index 4
     [invocation getArgument:&unretainedArgument atIndex:4];
     FIRAuthURLCallbackMatcher callbackMatcher = unretainedArgument;
+    NSMutableString *redirectURL =
+        [NSMutableString stringWithString:kFakeRedirectURLStringWithReCAPTCHAToken];
```

Comment:
Please also check that the URL is rejected by the callback matcher without the event ID by adding the following lines here:
```
// Verify that the URL is rejected by the callback matcher without the event ID.
XCTAssertFalse(callbackMatcher([NSURL URLWithString:redirectURL]));
// Verify that the URL is accepted by the callback matcher with the matching event ID.
```

Reply:
Done, thanks!